lib/drives/models/drive.dart

```diff
@@ -14,39 +14,39 @@ class Drive extends Trip {
 
   final List<Ride>? rides;
 
-  Drive({
-    super.id,
-    super.createdAt,
-    required super.start,
-    required super.startPosition,
-    required super.startTime,
-    required super.end,
-    required super.endPosition,
-    required super.endTime,
-    required super.seats,
-    this.cancelled = false,
-    required this.driverId,
-    this.driver,
-    this.rides,
-  });
+  Drive(
+      {super.id,
+      super.createdAt,
+      required super.start,
+      required super.startPosition,
+      required super.startTime,
+      required super.end,
+      required super.endPosition,
+      required super.endTime,
+      required super.seats,
+      this.cancelled = false,
+      required this.driverId,
+      this.driver,
+      this.rides,
+      super.hideinListView});
```

Comment:
please add the trailing comma here and everywhere else, this creates a lot of unnecessary whitespace changes and a worse format